src/ui/components/ExperimentBrowserPage.tsx

```diff
@@ -212,47 +212,34 @@ const ExperimentBrowserPage: FC = () => {
                       </option>
                     ))}
                   </Form.Select>
-                  {status === "Live" ? (
-                    <Dropdown>
-                      <Dropdown.Toggle className="option-button primary-fg py-2 my-1 mx-2 rounded small-font fw-bold grey-border light-bg">
-                        Actions
-                      </Dropdown.Toggle>
-                      <Dropdown.Menu>
-                        <Dropdown.Item
-                          onClick={() =>
-                            handleEnroll(
-                              experiment.id,
-                              selectedBranches[experiment.id],
-                            )
-                          }
-                        >
-                          Force Enroll
-                        </Dropdown.Item>
-                        <Dropdown.Item
-                          onClick={() =>
-                            handleGenerateTestIds(
-                              experiment.id,
-                              selectedBranches[experiment.id],
-                            )
-                          }
-                        >
-                          Generate Test IDs
-                        </Dropdown.Item>
-                      </Dropdown.Menu>
-                    </Dropdown>
-                  ) : (
-                    <Button
-                      className="option-button primary-fg py-0 my-1 mx-1 rounded small-font fw-bold grey-border light-bg"
-                      onClick={() =>
-                        handleEnroll(
-                          experiment.id,
-                          selectedBranches[experiment.id],
-                        )
-                      }
-                    >
-                      Force Enroll
-                    </Button>
-                  )}
+
+                  <Dropdown>
+                    <Dropdown.Toggle className="option-button primary-fg py-2 my-1 mx-2 rounded small-font fw-bold grey-border light-bg">
+                      Actions
+                    </Dropdown.Toggle>
+                    <Dropdown.Menu>
+                      <Dropdown.Item
+                        onClick={() =>
+                          handleEnroll(
+                            experiment.id,
+                            selectedBranches[experiment.id],
+                          )
+                        }
+                      >
```

Comment:
this will create a new function every render, causing a re-render of the entire subtree under it. We should use `useCallback` here.